.github/workflows/.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
```

Comment:
## [OPTIONAL]:

You have added test.md to your .gitignore. Great! But... Why is it still showing?
Git keeps a cache, and currently it stopped .test.md 

Here is a link explaining the solution!  Because you have fulfilled the requirement, I will approve this project. But, keep this in mind for the future. 😉 

https://stackoverflow.com/questions/9750606/git-still-shows-files-as-modified-after-adding-to-gitignore